RESTserver/src/main/java/com/softserve/if072/restservice/service/ShoppingListService.java

```diff
@@ -35,7 +36,7 @@ public ShoppingListService(ShoppingListDAO shoppingListDAO) {
 
     public ShoppingList getByUserAndProductId(int user_id, int product_id) throws DataNotFoundException {
         ShoppingList list = shoppingListDAO.getByUserAndProductId(user_id, product_id);
-        if (list != null) {
+        if (!CollectionUtils.isEmpty((Collection<?>) list)) {
```

Comment:
khm, what is this? Shopping list is a plain object, why do you cast it to the list?